package.json

```diff
@@ -43,37 +43,37 @@
     "@balena/contrato": "^0.6.0",
     "@balena/es-version": "^1.0.3",
     "@balena/lint": "^8.0.2",
-    "@types/bluebird": "^3.5.37",
-    "@types/chai": "^4.3.3",
-    "@types/chai-as-promised": "^7.1.5",
-    "@types/chai-like": "^1.1.1",
+    "@types/bluebird": "^3.5.42",
+    "@types/chai": "^4.3.14",
+    "@types/chai-as-promised": "^7.1.8",
+    "@types/chai-like": "^1.1.3",
     "@types/chai-things": "0.0.38",
-    "@types/common-tags": "^1.8.1",
+    "@types/common-tags": "^1.8.4",
     "@types/dockerode": "^3.3.28",
     "@types/event-stream": "^3.3.34",
-    "@types/express": "^4.17.14",
+    "@types/express": "^4.17.21",
     "@types/json-mask": "^2.0.3",
     "@types/JSONStream": "npm:@types/jsonstream@^0.8.33",
-    "@types/lodash": "^4.14.186",
-    "@types/memoizee": "^0.4.8",
+    "@types/lodash": "^4.17.0",
+    "@types/memoizee": "^0.4.11",
     "@types/mocha": "^10.0.6",
-    "@types/morgan": "^1.9.3",
-    "@types/node": "^20.11.24",
-    "@types/request": "^2.48.8",
-    "@types/rewire": "^2.5.28",
-    "@types/rwlock": "^5.0.3",
-    "@types/semver": "^7.3.12",
-    "@types/shell-quote": "^1.7.1",
+    "@types/morgan": "^1.9.9",
+    "@types/node": "^20.12.7",
+    "@types/request": "^2.48.12",
+    "@types/rewire": "^2.5.30",
+    "@types/rwlock": "^5.0.6",
+    "@types/semver": "^7.5.8",
+    "@types/shell-quote": "^1.7.5",
     "@types/sinon": "^17.0.3",
     "@types/sinon-chai": "^3.2.12",
     "@types/supertest": "^6.0.2",
     "@types/tar-stream": "^3.1.3",
-    "@types/webpack": "^5.28.0",
+    "@types/webpack": "^5.28.5",
     "@types/yargs": "^17.0.32",
     "balena-auth": "^6.0.1",
     "balena-register-device": "^9.0.2",
     "balena-request": "^13.3.1",
-    "blinking": "^0.0.7",
+    "blinking": "^0.0.8",
```

Comment:
This is purely to fix the use of a `.ts` file instead of a `.d.ts` for types. A `.d.ts` is used as-is whereas a `.ts` will have this repo's tsconfig.json applied (which doesn't go so well..)